package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nestjs-pino",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
releases are made separately from features. drop all changes in package*.json files